packages/react/src/primitives/Select/Select.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Flex } from '../Flex';
+import { View } from '../View';
+import { IconExpandMore } from '../Icon';
+import { SelectProps } from '../types';
+import { ComponentClassNames } from '../shared';
+
+export const Select: React.FC<SelectProps> = (props) => {
```

Comment:
I like this idea! I would like to build the `SelectField` in a separate PR with your `<Label>` component out of box to avoid squeezing too much in this one because things like testing, docs may also need to update as well